libs/core/src/lib/multi-input/multi-input.component.ts

```diff
@@ -466,13 +468,17 @@ export class MultiInputComponent implements
 
     /** @hidden */
     addOnButtonClicked(): void {
-        if (!this.open) {
-            this.handleSearchTermChange();
-        } else {
-            this.openChangeHandle(false);
-        }
+        this.openChangeHandle(!this.open);
     }
 
+    /** @hidden */
+    private _propagateSearchTermChange(): void {
+        this.searchTermChange.emit(this.searchTerm);
+        this.displayedValues = this.filterFn(this.dropdownValues, this.searchTerm);
```

Comment:
I'm not sure if we should have `displayedValues` updated in a method which (based on the name) should be responsible for propagating some changes.

Reply:
I changed naming